### DISCUSSION TITLE
Adjust the size of the right blank area, for Publication&Blog pages

### DISCUSSION THREAD

**Discussion by Eternal579** [academicpages/academicpages.github.io#2473] [Q&A]
I like this template so much, it's useful for expressing myself!
However, I got problems when i try to adjust the size of the right blank area.
<img width="1112" alt="微信图片_20240926123649" src="https://github.com/user-attachments/assets/581a7f40-7e0d-486e-bc8d-f3204aa5b756">
Is there any way to adjust this? 'cause my paper title is really long, having so much blank space on the right really bothers me.

**Comment by zhangyumeng1sjtu** [comment]
You can modify the variables`$right-sidebar-width-narrow`, `$right-sidebar-width`, and `$right-sidebar-width-wide` in the SCSS file `_sass/_variables.scss`.

**Comment by rjzupkoii** [comment]
This has been addressed in the latest version of the template, thanks to changes made by @HirokiHamaguchi.

**Comment by jakesong15** [comment]
I've not had success with changing the variables suggested by zhangyumeng1sjtu, but managed to find a work-around solution for this (YMMV):

- For pages where you want to **remove** the empty space on the right, begin the page files in your _pages directory with **layout: archive**
- For pages where you want to **insert** the empty space on the right, begin the page files in your _pages directory with **layout: single**
